apps/address-service/domains/common/utils/services/search/providers/utils/pullenti/settlementResolver.js

```
@@ -0,0 +1,38 @@
+    switch (gar.area?.type) {
+        case 'село':
+            settlement_type = 'с'
+            settlement_type_full = 'село'
+            break
+        case 'поселок':
+            settlement_type = 'пос'
+            settlement_type_full = 'поселок'
+            break
+        case 'деревня':
+        default:
+            settlement_type = 'д'
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)